run_api.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import os
+import shutil
+from fastapi import FastAPI, File, UploadFile, Depends, Response
+from fastapi.responses import FileResponse
+import uvicorn
+from pydantic import BaseModel
+from typing import Optional, Literal
+
+import roop.globals
+from roop.core import decode_execution_providers, suggest_execution_threads, limit_resources, start, \
+    get_frame_processors_modules, suggest_execution_providers
+from roop.utilities import resolve_relative_path
+
+
+app = FastAPI()
+
+class RoopModel(BaseModel):
+    frame_processor: Optional[list] = ['face_swapper']
```

Comment:
I like that you extended the typing... but we now have repeated code from globals.py

Reply:
Removed the Model class. Now all parameters are part of the method. Unfortunately, there is no certain way to avoid code duplication without changing a lot of root repo.